values.yaml

```diff
@@ -258,6 +258,10 @@ security:
     # Note: The user is created regardless of if the existingSecret is provided or not.
     username: provisioner
     password: iHaveSuperpowers
+    # Define an initial password for the provisioner user as a bcrypt hash.
+    # If the password is not specified, it will use the default one specified in the provisioner.password.
+    # Important: This value must be a bcrypt encrypted hash, not a plaintext string. The format is "{bcrypt}<the-encrypted-hash>"
```

Comment:
Make the "{bcrypt}" automatically be added to the hash in the user.js file so the user only has to input the hash. This improves the user experience.